OpenChange/RTFHandler.m

```diff
@@ -1437,20 +1450,17 @@ - (NSMutableData *) parse
         {
           c = *_bytes;
           // We avoid appending NULL bytes or endlines
-          if (c && (c != '\n')) 
+          if (c && (c != '\n') && (c != '\r'))
             {
-              if (c <= UTF8_FIRST_BYTE_LAST_CODEPOINT) 
```

Comment:
Now `UTF8_FIRST_BYTE_LAST_CODEPOINT` is no longer used. We should remove the `define`